engine/src/main/java/org/destinationsol/common/SolMath.java

```diff
@@ -15,49 +15,72 @@
  */
 package org.destinationsol.common;
 
-import java.util.ArrayList;
-
-import org.destinationsol.Const;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 import com.badlogic.gdx.math.MathUtils;
 import com.badlogic.gdx.math.Vector2;
 import com.badlogic.gdx.utils.JsonValue;
 import com.badlogic.gdx.utils.Pool;
+import org.destinationsol.Const;
+
+import java.util.ArrayList;
 
 /**
- * A class with helpful mathematical functions
+ * This class presents a collection of handy mathematical as well as programmatical functions, for working with angles,
+ * distances, speeds, vectors.
  */
 public class SolMath {
-    private static Logger logger = LoggerFactory.getLogger(SolMath.class);
+    /**
+     * The pi number constant.
+     */
     public static final float PI = MathUtils.PI;
+    /**
+     * Multiply radians by this number to convert them to degrees.
+     */
     public static float radDeg = MathUtils.radDeg;
+    /**
+     * Multiply degrees by this number to convert them to radians.
+     */
     public static float degRad = MathUtils.degRad;
-    public static Pool<Vector2> vs = new Pool<Vector2>() {
+    /**
+     * Stores {@link Vector2 vectors} used for borrowing, see {@link #getVec()} for more info.
+     */
+    private static Pool<Vector2> vectorPool = new Pool<Vector2>() {
         @Override
         protected Vector2 newObject() {
             return new Vector2();
         }
     };
-    public static int VECTORS_TAKEN;
-    public static Vector2 tmp = new Vector2();
+    /**
+     * Represents the amount of {@link Vector2 vectors} currently borrowed by {@link #getVec()}.
+     */
+    private static int vectorsTaken;
 
+    /**
+     * Converts boolean to integer, where {@code true} equals {@code 1} and {@code false} equals {@code -1}.
+     *
+     * @param b boolean to convert
+     * @return int representation of the boolean
+     */
     public static int toInt(boolean b) {
         return b ? 1 : -1;
     }
 
+    /**
+     * Returns absolute value of a float.
```

Comment:
Wat. This is literally a method that already exists. I don't know why this is here. But at least now it is documented! :-)

Same story as earlier, this is kinda progress until we throw it all away so 👍